### ISSUE TITLE
[Bug]: The second part of the image at the junction of pages is not displayed in PDF.

### ISSUE THREAD

**Issue by abvas** [puppeteer/puppeteer#9260]
### Bug description

If an image hits a page junction and splits into two parts, then the second (bottom) part of the image is not displayed in the generated PDF. Previous versions (eg 18.1.0) did not have this problem.

bad (19.2.2)
[https://s3.amazonaws.com/appforest_uf/f1668262049636x870986849179053800/doc_client.pdf](https://s3.amazonaws.com/appforest_uf/f1668262049636x870986849179053800/doc_client.pdf)

good (18.1.0):
[https://s3.amazonaws.com/appforest_uf/f1668266597423x541694432246403260/doc_client.pdf](https://s3.amazonaws.com/appforest_uf/f1668266597423x541694432246403260/doc_client.pdf)

### Puppeteer version

19.2.2

### Node.js version

v17.4.0

### npm version

8.4.1

### What operating system are you seeing the problem on?

Windows

### Configuration file

```JavaScript
const puppeteer = require('puppeteer');
async function getPdf() {
  let url = 'https://puppeteer.bubbleapps.io/version-test/test';

  let browser = await puppeteer.launch({ defaultViewport: null });
  let page = await browser.newPage();
  await page.goto(url, { waitUntil: ['load', 'networkidle0'] });
  await page.emulateMediaType('screen');

  const pdfConfig = {
    path: 'pdf.pdf',
    format: 'A3',
    printBackground: true,
    preferCSSPageSize: true,
  };
  await page.pdf(pdfConfig);
  await browser.close();
}
getPdf();
```

**Comment by Lightning00Blade** [comment]
@abvas We can see the issue happening and have filed a regression bug. You can follow it here -https://bugs.chromium.org/p/chromium/issues/detail?id=1383890

**Comment by OrKoN** [comment]
I believe this would be fixed with the next release 19.6.0 that includes the latest chromium binary.
